docs/workshop.md

```diff
@@ -0,0 +1,702 @@
+│   ├── AppFooter.vue
+│   ├── AppHeader.vue
+│   └── AppSidebar.vue
+├── locales
+│   ├── de.json
+│   └── en.json
+├── README.md
+├── static
+│   ├── app-conf.json
+│   ├── app-conf-minimal.json
+│   ├── app-conf-projected.json
+│   ├── app-conf-sidebar.json
+│   ├── css
+│   │   └── app.css
+│   ├── data
+│   │   ├── 2012_Earthquakes_Mag5.kml
+│   │   └── shops-dannstadt.geojson
+│   └── icon
+│       ├── circle.svg
+│       └── favicon.ico
+├── WguAppTemplate.vue
+└── WguApp.vue
+```
+
+For the moment we will look at the `static` directory. As the name implies, it contains static content like images, CSS files, (geo)data or configuration files. The configuration files are the central point to change the behavior of Wegue.
+
+The file `app-conf.json` controls the Wegue instance you can see in your browser. It defines the map layers that shall be seen and also defines which features of Wegue shall be used. The default config file looks overwhelming on first sight. That's why, we replace it with a minimal configuration file,
+
+Replace the content of `app-conf.json` with this:
+
+```json
+{
+
+  "colorTheme": {
+    "themes": {
+      "light": {
+        "primary": "#2E7D32"
+      }
+    }
+  },
+
+  "mapZoom": 2,
+  "mapCenter": [0, 0],
+
+  "mapLayers": [
+    {
+      "type": "XYZ",
+      "name": "Carto Positron",
+      "url": "https://basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png",
+      "lid": "positron",
+      "attributions": "Maps: © <a href='https://carto.com/attribution/' target='_blank'>CARTO</a> | <a href='https://openstreetmap.org/copyright' target='_blank'>© OpenStreetMap</a>-contributors | "
+    }
+  ]
+}
+```
+
+Open your browser that runs Wegue and refresh the page (typically clicking F5 on your keyboard). Now you should see this web map:
+
+![minimal webmap](_media/workshop/minimal-map.png)
+
+The current color is green and defined via a RGB-code. Change it to `#E0BBE4` (a kind of purple) or any other color you like. You can use a [color picker](https://www.w3schools.com/colors/colors_picker.asp) for finding your desired RGB-code. After a refresh of your browser tab, you should see the new color.
+
+![minimal webmap with purple color](_media/workshop/minimal-map-purple.png)
+
+When a user opens the map it will initially show the whole world. Let's say we would like to focus our map on Frankfurt in Germany. We would need to change the two properties `mapCenter` and `mapZoom`.
+
+The `mapCenter` is a coordinate in used map projection. In our case the default projection "web mercator" with the code `EPSG:3857` is used. For finding the coordinate you can use [this website](https://epsg.io/map#srs=3857&x=965552.541298&y=6466228.626299&z=11&layer=streets). The result might be `965552.541298 6466228.626299`. We must convert them to a JSON Array by surrounding them with bracket `[]` and separating them by a comma `,`. Additionally, We can safely remove the numbers behind the digit, because are unnecessarily precise.
```

Comment:
"The `mapCenter` is a coordinate in used map projection." => "The `mapCenter` is a coordinate in the projection of the map."

Reply:
fixed